src/components/UserInfo.js

```diff
@@ -1,16 +1,33 @@
 export default class UserInfo {
-  constructor(nameSelector, professionSelector) {
-    this._name = document.querySelector(nameSelector);
-    this._profession = document.querySelector(professionSelector);
+  constructor(profileNameSelector, profileProfessionSelector, profileImageSelector) {
+    this._name = document.querySelector(profileNameSelector);
+    this._profession = document.querySelector(profileProfessionSelector);
+    this._avatarEl = document.querySelector(profileImageSelector);
   }
   getUserInfo() {
     return {
       name: this._name.textContent,
       profession: this._profession.textContent,
     };
   }
-  setUserInfo(name, profession) {
-    this._name.textContent = name;
-    this._profession.textContent = profession;
+
+  setUserInfo({ userData }) {
```

Comment:
**Could be improved**

You can set all info here


`setUserInfo( { name, about, avatar, _id } )`

Reply:
Leaving comment open and taking improvement under advisement for future updates